rack-protection/rack-protection.gemspec

```diff
@@ -40,6 +40,8 @@ RubyGems 2.0 or newer is required to protect against public gem pushes. You can
 
   # dependencies
   s.add_dependency 'rack', '~> 2.2', '>= 2.2.4'
+  s.add_dependency 'base64'
```

Comment:
Can you put this line above the `rack` line? So the list is sorted alphabetically

Reply:
@dentarg 👍